PocketKit/Sources/SharedPocketKit/LastRefresh.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+
+public protocol LastRefresh {
+    var lastRefreshSaves: Double? { get }
+    var lastRefreshArchive: Double? { get }
+    var lastRefreshTags: Double? { get }
+    var lastRefreshHome: Double? { get }
```

Comment:
nit: because they are time intervals, for readability you could use the `TimeInterval` typeAlias here